Updated author names

waleediqbal411 opened the pull request:
Faculty Members added in Queen Mary University of London

# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted. Note that pull requests may take some time to get merged (please don't contact us for at least three months for updates, unless you are a sponsor - see below).

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT WILL BE SUMMARILY REJECTED.
You must read and check **all** the boxes below by filling them in with an X or your PR will be rejected.

**The Basics**

- [x] All pull requests and issues must come from non-anonymous accounts.

- [x] Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

- [x] Combine multiple updates to a single institution into a **single PR.**

- [x] Only submit one pull request per institution.

- [x] Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv` and `old/industry.csv` (see below).

- [x] Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use the GitHub user interface or a text editor like emacs or NotePad instead.

- [x] Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files. **Do not modify `csrankings.csv`, which is auto-generated.**

- [x] Check to make sure that you have no spaces after commas, or any missing fields.

- [x] Check to make sure the home page is correct.

- [x] Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`).

- [x] Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

- [x] If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

**Inclusion criteria**

- [x] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. **Provide justification, pointing to specific home pages
showing how faculty not in a CS department meet the inclusion criteria,
e.g. showing a courtesy appointment in CS.** Faculty must also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [x] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [x] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [x] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [x] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [x] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

waleediqbal411 committed:
Updated author names

Faculty Members added in Queen Mary University of London

waleediqbal411 committed:
Update author list Queen Mary University of London

Faculty Members added in Queen Mary University of London

waleediqbal411 committed:
Update authors Queen Mary University of London

Update authors Queen Mary University of London

emeryberger commented:
Please use the actual faculty home pages (not the departmental pages).

waleediqbal411 committed:
Two Faculty Members Updated at Queen Mary University of London

Two Faculty Members Updated at Queen Mary University of London

waleediqbal411 commented:
Faculty home pages are updated now.

emeryberger commented on csrankings-i.csv:
```diff
@@ -368,10 +369,10 @@ Itaru Kitahara,University of Tsukuba,https://sites.google.com/image.iit.tsukuba.
 Ittay Eyal,Technion,http://webee.technion.ac.il/people/ittay,1oUGY7cAAAAJ
 Iulian Neamtiu,NJIT,https://web.njit.edu/~ineamtiu,8qU-5YMAAAAJ
-Ivan Aaen,Aalborg University,http://people.cs.aau.dk/~ivan,-8-9oRgAAAAJ
+Ivan Aaen,Aalborg University,http://people.cs.aau.dk/~ivan,89oRgAAAAJ
```
Your editor apparently deleted the leading hyphen from this and the below Google Scholar IDs. I'd be interested to know how this happened so we can prevent it from happening in the future!

emeryberger commented on csrankings-i.csv:
```diff
@@ -368,10 +369,10 @@ Itaru Kitahara,University of Tsukuba,https://sites.google.com/image.iit.tsukuba.
-Ivan Bjerre Damgård,Aarhus University,https://cs.au.dk/~ivan/,-UhqXIEAAAAJ
-Ivan Damgård,Aarhus University,https://cs.au.dk/~ivan/,-UhqXIEAAAAJ
+Ivan Bjerre Damgård,Aarhus University,https://cs.au.dk/~ivan/,UhqXIEAAAAJ
+Ivan Damgård,Aarhus University,https://cs.au.dk/~ivan/,UhqXIEAAAAJ
```
Broken ID, see above.

emeryberger commented on csrankings-i.csv:
```diff
@@ -380,7 +381,7 @@ Ivan Herman,CWI,http://homepages.cwi.nl/~ivan,ofRrImEAAAAJ
 Ivan I. Garibay,University of Central Florida,https://www.cs.ucf.edu/~garibay,rPuLfoMAAAAJ
 Ivan Kisel,Goethe University Frankfurt,https://www.fias.science/de/computerwissenschaften/gruppen/ivan-kisel,NOSCHOLARPAGE
-Ivan Laptev,Ecole Normale Superieure,https://www.di.ens.fr/~laptev,-9ifK0cAAAAJ
+Ivan Laptev,Ecole Normale Superieure,https://www.di.ens.fr/~laptev,9ifK0cAAAAJ
```
Broken ID, see above.

emeryberger commented on csrankings-i.csv:
```diff
@@ -407,5 +408,5 @@ Iyad A. Kanj,DePaul University,http://www.cdm.depaul.edu/about/Pages/People/facu
 Iyad Katib,King Abdulaziz University,http://iyadkatib.com,DN3hbOsAAAAJ
 Iyad Ouaiss,Lebanese American University,http://services.sea.lau.edu.lb/faculty/iouaiss,Bs47qDUAAAAJ
-Izidor Gertner,CUNY,https://www.ccny.cuny.edu/profiles/izidor-gertner,-3lkqowAAAAJ
+Izidor Gertner,CUNY,https://www.ccny.cuny.edu/profiles/izidor-gertner,3lkqowAAAAJ
```
Broken ID, see above.

waleediqbal411 replied:
It was a regex mistake done by the author. It is fixed in the current push.

waleediqbal411 committed:
Two Faculty Members Updated at Queen Mary University of London

Two Faculty Members Updated at Queen Mary University of London

emeryberger commented:
"Teaching Fellow" sounds like non tenure-track faculty. Please clarify.

waleediqbal411 commented:
Yes. Teaching Fellow is not a Tenue-Track faculty position. It is primarily focused on Teaching and Scholarship in UK.

emeryberger commented:
Emphasis mine below:

> Eligible faculty include only full-time, *tenure-track* research
faculty members on a given campus who can solely advise PhD students in
Computer Science.

waleediqbal411 commented:
Thank you very much for your clarification. I have excluded the Teaching Fellow faculty update and kept only the tenure-track faculty update.

waleediqbal411 committed:
One Faculty Member added in Queen Mary University of London

One Faculty Member, Dr. Ignacio Castro added in Queen Mary University of London
